test/omaha/omaha_api_test.rb

```diff
@@ -94,6 +96,12 @@ def app
 
   def setup
     TestForemanClient.clear_requests
+    @contentpath = Dir.mktmpdir
+    Proxy::Omaha::Plugin.load_test_settings(
+      {
+        :contentpath => @contentpath
+      }
+    )
```

Comment:
I think you can simplify it:
```suggestion
    Proxy::Omaha::Plugin.load_test_settings(contentpath: @contentpath)
```